examples/indexer-client/index.ts

```diff
@@ -0,0 +1,20 @@
+import { createIndexerClient } from "@latticexyz/store-sync/trpc-indexer";
+const indexer = createIndexerClient({
+  url: "http://localhost:3001",
+});
+
+const result = await indexer.findAll.query({
+  chainId: 31337,
+  address: "0x5FbDB2315678afecb367f032d93F642f64180aa3",
+});
+
+const blockNum = result.blockNumber;
+console.log(`Block number: ${blockNum}`);
```

Comment:
```suggestion
const blockNumber = result.blockNumber;
console.log(`Block number: ${blockNumber}`);
```

nit, but we generally like to avoid abbreviations whenever possible.